### PR TITLE
refactor: Remove single mode

### LINKED ISSUE #1764: Remove single node batch query
For example, `QueryManager#schedule_single()`  or `RW_DIST_QUERY=1` etc...

Do you think it's ok to do this now? cc @liurenjie1024

### PR DESCRIPTION
**Pull request by liurenjie1024**
## What's changed and what's your intention?

RT

## Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests

## Refer to a related PR or issue link (optional)

Closes #1764

### CODE REVIEW

**Review by BugenZhao — APPROVED** [review]
Cool!

File: src/frontend/src/handler/mod.rs [hunk 1]
```diff
@@ -24,13 +24,13 @@ pub mod create_mv;
 pub mod create_source;
 pub mod create_table;
 mod describe;
+pub mod dml;
 pub mod drop_mv;
 pub mod drop_table;
 mod explain;
 mod flush;
 #[allow(dead_code)]
```
Comment (skyzh) [inline src/frontend/src/handler/mod.rs:32]: remove allow dead code?

**Review by skyzh — APPROVED** [review]

**Comment by skyzh** [comment]
It seems that the CLA assistant is not stable enough 🤣

File: src/frontend/src/handler/mod.rs (hunk 1 above)
Comment (liurenjie1024) [reply src/frontend/src/handler/mod.rs:32]: I tried but seems we still have some missing part. Let's remove them later.

**Pull request merged**